### ISSUE TITLE
Why isn't the Realsense T265 configuration available in /templates/complete.py

### ISSUE THREAD

**Issue by TCIII** [autorope/donkeycar#692]
I am running DC 3.1.5 on a NVIDIA Nano 4GB and would like to integrate my Realsense T265 into the DC software. However, at the moment there appears to be a CAMERA_TYPE only available for the Realsense D435i in /templates/complete.py.

All of the DC python support programs (/templates/path_follow.py, /templates/cfg_path_follow.py, /parts/realsense2.py, and /parts/path.py) appear to be available to support the T265.

Is there a way to configure the DC configuration to use the Realsense T265?

**Comment by TCIII** [comment]
I have found that a version of DC can be built for the Realsense T265 tracking camera using the template "path_follow". However the "manage.py" code that is built attempts to import "pigpio" which does not exist for DC when built for the NVIDIA Nano instead it is jetson-gpio. 
I downloaded the DC master files and used "pip install -e .[nano]" to install the Nano version of the files and not the Rpi version.
Comments?
